source/docs/contribute/documentation.md

```diff
@@ -68,6 +68,19 @@ description: "A summary of the contents of this page"
 ---
 ```
 
+## About screenshots

```

Comment:
With this moving here, the `source/assets/images/screenshots/README.md` file should be removed.